packages/cli/package.json

```diff
@@ -67,7 +67,7 @@
     "ws": "^1.1.0"
   },
   "peerDependencies": {
-    "react-native": "^0.61.0"
+    "react-native": "^0.62.0"
```

Comment:
Why don't we do `|| 0.62.0-rc.3` or similar to remove warnings for RC versions as well? I don't think this would hurt.

Reply:
Simple answer would be that we just don't care, which is fine as well

Reply:
Ok, went with including RC as well because we support it after all